### ISSUE TITLE
Support "Service Bus for Windows 1.1" in Azure WebJobs

### ISSUE THREAD

**Issue by RonakThakkar** [Azure/azure-webjobs-sdk#615]
Microsoft.Azure.WebJobs.ServiceBus simplifies a lot of Azure Service Bus programming. We would appreciate if Microsoft.Azure.WebJobs.ServiceBus also supports "Service Bus for Windows 1.1"

Microsoft.Azure.WebJobs.ServiceBus Nuget Package has dependency on  WindowsAzure.ServiceBus >= 2.3.1
But as per the documentation of "Service Bus for Windows 1.1", WindowsAzure.ServiceBus 2.1 is only compatible with "Service Bus for Windows 1.1"

**Background**
Basically we are developing application that should support deployment on Cloud and On-premise. In case of cloud we use Azure Service Bus and in case of On-premise, we use "Service Bus for Windows 1.1"

**Comment by rustd** [comment]
We are only supporting Azure ServiceBus.
